### PR TITLE
[Staging] Create "Footer" component

### PR DESCRIPTION
**Pull request by kilemensi**
## Description

Implement the site Footer per designs.

Fixes # 164864420

## Type of change

- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Design

Screenshot_2019-03-27 Zeplin - Project

#### Desktop

Screenshot_2019-03-27 React App

#### Mobile

Screenshot_2019-03-27 React App(1)

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation

### CODE REVIEW

File: takwimu/takwimu_ui/src/components/Footer/QuickLinks.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  { href: '/contact-us', label: 'Contact Us' }
+];
+const LEGAL = [
+  { href: '/legal', label: 'Terms of use' },
+  { href: '/legal', label: 'Privacy Policy' }
+];
+function QuickLinks({ classes }) {
+  return (
+    <div className={classes.root}>
+      <Title className={classes.text}>Quick Links</Title>
```
Comment (karimkawambwa) [inline takwimu/takwimu_ui/src/components/Footer/QuickLinks.js]: minor nitpick, title is already white

**Comment by nerdlet** [comment]
@karimkawambwa Maybe two changes I noticed 
1. The logo images are a bit blurry for the footer 
image

2. Text description content/list content colour is white, which is a bit different from the design

**Comment by kilemensi** [comment]
> LGTM But...
> 
> The footer gets misplaced at a certain dimension, think xl
> 
> Screen Shot 2019-03-27 at 2 56 05 PM
> 
> Could you add bottom padding or height to `where to next` in this pr so it doesn't touch the footer.

I can't replicate this @karimkawambwa, even on dobule screens (>3500px width).
Screenshot_2019-03-27 React App(1)

**Comment by kilemensi** [comment]
> @karimkawambwa Maybe two changes I noticed
> 
>     1. The logo images are a bit blurry for the footer
>        image
> 
>     2. Text description content/list content colour is white, which is a bit different from the design

 1. I think setting size in `rem` causes images to be resized and hence blurry. I've removed the sizes and update the screenshot.
 2. I'm not sure what you mean here @nerdlet. From the design, the text is suppose to be **white**.
Screenshot from 2019-03-27 16-22-59

**Comment by nerdlet** [comment]
@kilemensi My bad...used the image of the design on this PR ....double checked with the design...Just ignore it

**Comment by karimkawambwa** [comment]
@kilemensi weird, if it works then merge. I was using the chrome browser.

**Pull request merged**